src/components/QuickOpenModal.js

```diff
@@ -128,7 +128,10 @@ export class QuickOpenModal extends Component<Props, State> {
     let results = functions;
     if (this.isVariableQuery()) {
       results = variables;
+    } else {
+      results = results.filter(result => result.title !== "anonymous");
     }
```

Comment:
this might benefit from a unit test in QuickOpenModal.spec.js

Reply:
Done